### ISSUE TITLE
Encoding problems with plots in R Presentation

### ISSUE THREAD

**Issue by molx** [yihui/knitr#1061]
Hello,

I'm having trouble using special characters in RPresentation plots. For example:

    Test
    ========================================================
    author: 
    date: 

    First Slide
    ========================================================

    ```{r}
    plot(1:10, main="Título áéíóúãõ")
    ```

Will produce the following:

![plotrpress](https://cloud.githubusercontent.com/assets/693435/8297602/4cb0373c-1935-11e5-9a93-55c374b80203.png)

The encoding problem only happens with plots. Code and regular text are displayed properly. I apologized if this isn't an knitr problem - I'm not sure on what saves the plots from presentations.

```
> sessionInfo()
R version 3.2.0 (2015-04-16)
Platform: x86_64-w64-mingw32/x64 (64-bit)
Running under: Windows 7 x64 (build 7601) Service Pack 1

locale:
[1] LC_COLLATE=Portuguese_Brazil.1252  LC_CTYPE=Portuguese_Brazil.1252    LC_MONETARY=Portuguese_Brazil.1252
[4] LC_NUMERIC=C                       LC_TIME=Portuguese_Brazil.1252    

attached base packages:
[1] stats     graphics  grDevices utils     datasets  methods   base     

loaded via a namespace (and not attached):
[1] tools_3.2.0
```

**Comment by yihui** [comment]
Your session info did not show the version of knitr. Have you tried FAQ 1? http://bit.ly/knitr-faq

**Comment by molx** [comment]
The knitr version is `knitr_1.10.5`. I checked the FAQ, I just didn't try StackOverflow, even though I'm an active user, since I'm under the impression it's some Rpres/kntir issue. Unfortunately the Rpres documentation (specially the bits which don't match RMarkdown's) isn't very extensive.

**Comment by yihui** [comment]
Do R Markdown documents work then?

**Comment by molx** [comment]
Yes, Rmarkdown works fine without any extra configuration:

![plotrpress2](https://cloud.githubusercontent.com/assets/693435/8344880/9b4dfcc4-1abd-11e5-8262-d4f6db958cb1.png)

**Comment by yihui** [comment]
So it seems to be specific to Rpres. I have no idea how this could possibly happen...

**Comment by molx** [comment]
That's a bummer. I noticed it also displays incrrect characters in two other situations:

 - Error mesages (which are in portuguese, but the non-ascii characters are bugged)
 - Variable names with special characters. Not that I actually use them, but I did by mistake and received an error, while generally it works.

All this is specific to Rpres. If it isn't knitr's fault, is there anywhere I can submit an issue to Rpres? I'm not sure if RStudio repo would be fit, I didn't find any information on Rpres there.
